### PR TITLE
Added MAPI basic auth key

### PR DESCRIPTION
**Pull request by yolandeleungFT**

### CODE REVIEW

File: ansible/userdata/default_instance_user_data.yaml [hunk 1]
```diff
@@ -100,6 +100,7 @@ coreos:
           ExecStart=/bin/sh -c "etcdctl mk /ft/config/kafka-bridge/ucs_queue_proxy_addrs '{{bridging_message_queue_proxy}}' > /dev/null 2>&1 || true;"
           ExecStart=/bin/sh -c "etcdctl mk /ft/_credentials/wordpress/content_api_key '{{ wordpress_api_key }}' > /dev/null 2>&1 || true;"
           ExecStart=/bin/sh -c "etcdctl mk /ft/_credentials/logstash-forwarder/cert '{{ logstash_forwarder_crt }}' > /dev/null 2>&1 || true;"
+          ExecStart=/bin/sh -c "etcdctl set /ft/_credentials/methode-api/authorization_key '{{ methode_api_proxy_authorization_key }}' > /dev/null;"
```
Comment (Zapadlo) [inline ansible/userdata/default_instance_user_data.yaml]: You probably want to copy the pattern from the other entries above ^^

Comment (Zapadlo) [inline ansible/userdata/default_instance_user_data.yaml]: You probably want to copy the pattern from the other entries above ^^

Comment (yolandeleungFT) [inline ansible/userdata/default_instance_user_data.yaml]: oops. done.

**Comment by Zapadlo** [comment]
+1

**Pull request merged**